### DISCUSSION TITLE
다른 주소로 잘못 입금한 경우 거래 취소가 가능한가요?

### DISCUSSION THREAD

**Discussion by ByeongHoChoi** [blockscout/blockscout#9795] [Q&A]
NFT를 구매하는 과정에서 프로필에 있는 주소로 입금했는데 취소 가능한가요?

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.

**Comment by ByeongHoChoi** [comment]
체인지원팀 연락방법을 가르쳐 주세요내 Galaxy에서 보냄
-------- 원본 이메일 --------발신: Maxim Filonov ***@***.***> 날짜: 24/4/3  18:02  (GMT+09:00) 받은 사람: blockscout/blockscout ***@***.***> 참조: BHChoi ***@***.***>, Author ***@***.***> 제목: Re: [blockscout/blockscout] 다른 주소로 잘못 입금한 경우 거래 취소가 가능한가요? (Discussion #9795) 
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
